src/useAsObservableSource.ts

```diff
@@ -8,6 +8,15 @@ export function useAsObservableSourceInternal<TSource>(
     usedByLocalStore: boolean
 ): TSource {
     const culprit = usedByLocalStore ? "useLocalStore" : "useAsObservableSource"
+    if (process.env.NODE_ENV !== "production" && usedByLocalStore) {
```

Comment:
I believe this is there so that tests can assert dev errors. (there's `global.__DEV__ = false` in a `jest.setup.js` file)

I'll change to `__DEV__` and make sure dev tests set that correctly